source/extensions/transport_sockets/alts/tsi_socket.cc

```diff
@@ -327,18 +313,32 @@ Network::IoResult TsiSocket::repeatProtectAndWrite(Buffer::Instance& buffer, boo
 }
 
 Network::IoResult TsiSocket::doWrite(Buffer::Instance& buffer, bool end_stream) {
+  Network::IoResult result = {Network::PostIoAction::KeepOpen, 0, false};
```

Comment:
nit: result is never used without it being written by a raw_buffer_socket_->doWrite.  Consider removing this local variable and instead declaring it when doWrite is called.

Network::IoResult result = raw_buffer_socket_->doWrite(...);

Reply:
Done.